blog/2022-08-30-q-and-a-august-2022-first-part.mdx

```diff
@@ -134,17 +134,17 @@ Luos uses something we call "HAL": [hardware attraction layer](/docs/luos-techno
 
 Yes, Luos could use Nuttx as an HAL for Luos. We could access the physical resources available on the board using POSIX interfaces. This could allow Luos to access all the boards supported by Nuttx.
 
-Embedded system developers didn't use to care about a project's API. But we think that today their points of view are evolving toward using this kind of standards, because everyone is using it. It allows them to create developments that will be reusable extremely more easily.
+Embedded system developers didn't use to care about a project's API. But we think their point of view is evolving today towards using these standards because everyone is using them. It allows them to create developments that will be reusable extremely easily.
 
 Historically, Luos was thought to work for bare metal applications only. That is why we don't have a lot of usages, use cases, and examples using a real-time operating system.
 
 We did an experiment using [FreeRTOS](/tutorials/freertos/intro) just a few months ago: it that was easy for us to adapt it to work with FreeRTOS.
 
-Luos was then ready to multithreading. Although this is something new to us, we plan to develop a lot of support for different kinds of real-time operating systems in the future, so NuttX is on top of the list right now, thanks to Alan. We will work on this kind of subject during the coming months.
+Luos is now ready to multithread. Although this is something new to us, we plan to develop a lot of support for different kinds of real-time operating systems in the future, so NuttX is on top of the list right now, thanks to Alan. We will work on this kind of subject during the coming months.
```

Comment:
Maybe add a link or something related to Alan, since we are mentionning a name ?

Reply:
I will add his Discord nickname